packages/opensrp-plans/package.json

```diff
@@ -52,5 +52,8 @@
     "@opensrp/plan-form-core": "^0.0.1",
     "@opensrp/react-utils": "^0.0.1",
     "@opensrp/server-service": "^0.0.12"
+  },
+  "devDependencies": {
+    "@turf/turf": "^5.1.6"
```

Comment:
What will we be using this for?

Reply:
Geometry type in jurisdiction Dux

Reply:
let me replace this with geojson, that already exists in the repo, and its only types.